docs/user/README.md

```diff
@@ -32,12 +39,12 @@ Currently, logs are based on the Fluent Bit protocol. If you're curious about th
 
 ## Components
 
-![Components](./assets/general-components.drawio.svg)
+![Components](./assets/telemetry-components.drawio.svg)
 
 ### Telemetry Manager
 
-The module ships the Telemetry Manager as its core component. The manager implements the Kubernetes controller pattern and manages the whole lifecycle of all other components covered in the module. The manager watches for LogPipeline, TracePipeline, and MetricPipeline resources created by the user. With these, the user describes in a declarative way what data of a signal type to collect and where to ship it.
-If the operator detects a configuration, it rolls out the relevant collector components on demand.
+Kyma's Telemetry module ships the Telemetry Manager as its core component. The manager implements the Kubernetes controller pattern and manages the whole lifecycle of all other components covered in the module. The manager watches for LogPipeline, TracePipeline, and MetricPipeline resources created by the user. With these, the user describes in a declarative way what data of a signal type to collect and where to ship it.
+If the manager detects a configuration, it rolls out the relevant collector components on demand.
```

Comment:
pls confirm, it's called manager now, not operator? or are those two different entities?